graphrag/query/cli.py

```diff
@@ -129,7 +129,9 @@ def _infer_data_dir(root: str) -> str:
     output = Path(root) / "output"
     # use the latest data-run folder
     if output.exists():
-        folders = sorted(output.iterdir(), key=os.path.getmtime, reverse=True)
+        expr = re.compile(r"\d{8}-\d{6}")
+        filtered = [f for f in output.iterdir() if f.is_dir() and expr.match(f.name)]
```

Comment:
What if you add another condition `and not f.name.startswith(".")` to guard against dot files in case the regex is not enough?

Reply:
I added another test folder to confirm that the regex is enough